server/src/main/java/org/apache/cloudstack/consoleproxy/ConsoleAccessManagerImpl.java

```diff
@@ -489,6 +494,6 @@ public String getConfigComponentName() {
 
     @Override
     public ConfigKey<?>[] getConfigKeys() {
-        return new ConfigKey[] { ConsoleProxyExtraSecurityValidationEnabled };
+        return new ConfigKey[] { };
```

Comment:
if so, we can remove `Configurable` from ConsoleAccessManager

Reply:
agree @nvazquez if there's no need for Configurable in this class.

Reply:
Done, thanks